### DISCUSSION TITLE
How to bypass the default append of v2 , if we have private registry

### DISCUSSION THREAD

**Discussion by Sam0508-SDE** [eclipse/jkube#3079] [Q&A]
We in our project have our project specific registry for image storage and fetch which is not exposed outside , what we noticed that plugin automatically appends a v2 at end of domain which is not valid for our registry , how can we bypass this behaviour

**Comment by manusa** [comment]
Could you provide more details about your project and your registry?

`v2` is likely being grabbed from the HTTP requests that JKube performs either using Jib or the [Docker HTTP protocol](https://docs.docker.com/docker-hub/api).

Without having more information, I'd say that this is caused because you are using a very old container image registry. Or because its implementation might not be according to the v2 spec.

If you provide more information, maybe we can determine the cause. Other than that, if the `v2` comes from where I suspect, there's little we can do from our side because that is part of the spec.
